frontend/src/metabase/syncing/containers/SyncProgress/SyncProgress.jsx

```diff
@@ -0,0 +1,8 @@
+import { connect } from "react-redux";
+import _ from "underscore";
+import { getUserIsAdmin } from "metabase/selectors/user";
+import SyncProgress from "../../components/SyncProgress";
+
+export default _.compose(
+  connect(state => ({ isAdmin: getUserIsAdmin(state) })),
```

Comment:
Are we good with showing it to all admins and not only the one who connected the database?

Reply:
No, we are not, but there is no BE field to check. It should be added soon.